src/ArchitectNow.Web.Mongo/StartupSample.cs

```diff
@@ -83,31 +66,32 @@ private JwtSigningKey ConfigureSecurityKey(JwtIssuerOptions issuerOptions)
 
         public void Configure(
             IApplicationBuilder app,
-            IApplicationLifetime appLifetime,
-            IAntiforgery antiforgery,
-            IConfiguration configuration)
+            IHostApplicationLifetime appLifetime)
         {
             _logger.LogInformation($"{nameof(Configure)} starting...");
 
             //Add custom middleware or use IStartupFilter
-            app.UseSwaggerUi3(settings =>
-            {
-                settings.GeneratorSettings.Title = "API";
-                settings.GeneratorSettings.Description = "API";
-                settings.DocumentPath = "/app/docs/v1/swagger.json";
-                settings.Path = "/app/docs";
-                settings.GeneratorSettings.DefaultEnumHandling = EnumHandling.String;
-                settings.GeneratorSettings.DefaultPropertyNameHandling = PropertyNameHandling.CamelCase;
-                settings.GeneratorSettings.Version = Assembly.GetEntryAssembly().GetName().Version.ToString();
-            });
+            // app.UseSwaggerUi3(settings =>

```

Comment:
All swagger code has been commented from every `StartupSample` code. Just to avoid an annoying msg on the compiler. This doesn't have any impact on any library since these files are not used anywhere and are there just as a reference.
![image](https://user-images.githubusercontent.com/21264330/100466670-07e8b280-30b0-11eb-81d9-0f3ad5bf8abe.png)